view/adminhtml/ui_component/snowmenu_menu_list.xml

```diff
@@ -36,6 +36,29 @@
             </item>
         </argument>
     </dataSource>
+    <listingToolbar name="listing_top">
+        <paging name="listing_paging"/>
+        <massaction name="listing_massaction">
+            <action name="enable">
+                <argument name="data" xsi:type="array">
+                    <item name="config" xsi:type="array">
+                        <item name="type" xsi:type="string">enable</item>
+                        <item name="label" translate="true" xsi:type="string">Enable</item>
+                        <item name="url" xsi:type="url" path="snowmenu/menu/massEnable/"/>
```

Comment:
I am not sure it makes a difference but you have extra slash at the end of path.

Reply:
It does not but I removed it anyway so it's consistent with other urls.